src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -2,8 +2,8 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { ActivatedRoute, ActivatedRouteSnapshot, Router } from '@angular/router';
 import { FormGroup } from '@angular/forms';
 import 'rxjs/add/operator/do';
-import { Jurisdiction, CaseState, CaseTypeLite, WorkbasketInputModel } from '../../domain';
-import { JurisdictionService, AlertService, WindowService, OrderService, WorkbasketInputFilterService } from '../../services';
+import { Jurisdiction, CaseState, CaseTypeLite, WorkbasketInputModel, CaseField } from '../../domain';
+import { JurisdictionService, AlertService, WindowService, OrderService, WorkbasketInputFilterService, FieldsUtils } from '../../services';
```

Comment:
Same thing with the import ordering and lint not being a happy 🐇 .

Reply:
As mentioned earlier - I ran yarn lint - no errors.